powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/serialization/AttestationObjectDeserializer.java

```diff
@@ -52,15 +52,16 @@ public AttestationObjectDeserializer(Class<?> vc) {
     }
 
     @Override
-    public AttestationObject deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException, JacksonException {
+    public AttestationObject deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException {
         try {
             final String originalTextValue = jsonParser.getText();
             final byte[] decodedAttestationObject = Base64.getDecoder().decode(originalTextValue);
             final AttestationObject attestationObject = cborMapper.readValue(decodedAttestationObject, AttestationObject.class);
             attestationObject.setEncoded(originalTextValue);
             return attestationObject;
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            logger.warn(e.getMessage(), e);
```

Comment:
👀 The declaration is `public abstract T deserialize(JsonParser p, DeserializationContext ctxt)
        throws IOException, JacksonException;` so why to catch the exception at all?

Reply:
We want to avoid a 500 error, but I will review this.

Reply:
I will leave the implementation as is for now, because I do not think a generic error handler for `IOException` is a good idea and we want to avoid unhandled 500 errors. We can improve it later in case of any issues, however this is a corner case which should not occur in actual implementation because the attestation object is provided by the authenticator and shouldbe serialized correctly.